src/index.html

```diff
@@ -11,8 +12,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <!-- #endregion -->
+  <body class="page">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Apple iMac 27 Retina 5K Monoblock"
+        class="card__image"
+      />
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__raiting">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <span class="card__reviews">Reviews: 5</span>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-value">$2,199</p>
+      </div>
+      <a
```

Comment:
I think it would be more readable, but it's up to u)
```suggestion
      </div>
      
      <a
```